ui/panels-plugin/src/plugins/time-series-chart/utils/test.json

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
Is this test.json being used anywhere? Can maybe be deleted, also weird how it got formatted this way, might be missing a comma or curly brace somewhere

Reply:
Haha omigod this is a total mistake, I was playing with something else and this has nothing to do with anything

(will delete!)